### PR TITLE
Add header.html file

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: bin/import-ngeo-apps [hunk 1]
```diff
@@ -136,6 +136,15 @@ def main():
                 ),
                 data,
             )
+            data = _sub(
+                re.escape("<ng-include src=\"'desktop/header.html'\"></ng-include>"),
+                "<ng-include src=\"'{entry_point}static/{version}/header.html'\"></ng-include>".format(
```
Comment (arnaud-morvan) [inline bin/import-ngeo-apps:141]:
```suggestion
                "<ng-include src=\"'{entry_point}static/{cache_version}/header.html'\"></ng-include>".format(
```

File: bin/import-ngeo-apps [hunk 2]
```diff
@@ -136,6 +136,15 @@ def main():
...
+                    entry_point="<%= htmlWebpackPlugin.options.vars.entry_point %>",
+                    version="<%= htmlWebpackPlugin.options.vars.version %>",
```
Comment (arnaud-morvan) [inline bin/import-ngeo-apps:143]:
```suggestion
                    cache_version="<%= htmlWebpackPlugin.options.vars.cache_version %>",
```

**Review by arnaud-morvan — APPROVED** [review]
Minor comment, not sure, but it is `cache_version` for favicon also in static folder.

File: bin/import-ngeo-apps (hunk 2 above)
Comment (sbrunner) [reply bin/import-ngeo-apps:143]: +1 thanks :-)

**Pull request merged**